boards/seeeduino_xiao/Kconfig

```diff
@@ -0,0 +1,23 @@
+# Copyright (c) 2020 HAW Hamburg
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+
+config BOARD
+    default "seeeduino_XIAO" if BOARD_SEEEDUINO_XIAO
+
+config BOARD_SEEEDUINO_XIAO
+    bool
+    default y
+    select CPU_MODEL_SAMD21G18A
+    select HAS_HIGHLEVEL_STDIO
+    select HAS_PERIPH_ADC
+    select HAS_PERIPH_I2C
+   # select HAS_PERIPH_PWM not implemented yet
```

Comment:
just remove that line, PWM is implemented for the CPU, it just lacks the board configuration.